draft-ietf-core-multipart-ct.md

```diff
@@ -56,7 +56,7 @@ informative:
 This memo defines application/multipart-core, an
 application-independent media-type that can be used
 to combine representations of several different media types into a single
-CoAP message-body with minimal framing overhead, each along with a CoAP
+message, typically a CoAP request or response body, with minimal framing overhead, each along with a CoAP
```

Comment:
I would s/typically/such as/ here and below.

Reply:
done in e65c47d